### PR TITLE
readme changed

### PR DESCRIPTION
**Pull request by andromash**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 This is the project that implements idea of secured internet shop. 
 The main subject of the shop is different beverages.
```
Comment (AnaLanda) [inline README.md]:
```suggestion
This is the project that implements the idea of an internet shop specializing in beverages. 
```

File: README.md [hunk 2]
```diff
@@ -3,6 +3,7 @@
 This is the project that implements idea of secured internet shop. 
 The main subject of the shop is different beverages. 
 This projects represents client - server architecture.
+Also this projects was build on N-tier architecture. So we have DAO layer, Service layer, Controller.
```
Comment (AnaLanda) [inline README.md]:
```suggestion
This project, which represents the client-server architecture, was build following the N-tier architecture and has a DAO layer, a Service layer, and Controllers. I built this project with SOLID principles and RBAC in mind. The client side of this application was created with JSP-pages and Bootstrap.
```

Comment (AnaLanda) [reply README.md]: I think you should move the 2 sentences from under the 'Technologies' list here.

Also, edit punctuation here:
`Technologies used in the project:`

File: README.md [hunk 3]
```diff
@@ -43,4 +44,9 @@ login: testAdmin
 password: 1111
 
 But first of all, there's file init_db.sql which holds sql script for creating appropriate DB.
-So run this code in MySQL on your machine first
+So run this code in MySQL on your machine first. You need to change name and password
+in ConnectionUtil so this code will run on your MySQL Server.
+Also this projects runs in Apache Tomcat. So you need to configure it.
+Maven is used as packaging tool, so you need to enable import all of the dependencies and plugins.
```
Comment (AnaLanda) [inline README.md]:
```suggestion
To launch the project, download the project onto your machine and make sure you have MySQL Workbench on it. The file `init_db.sql` holds the SQL script for creating the DB for this project - you need to run the code from this file in MySQL Workbench first. Then you need to change the name and password
in `ConnectionUtil` so this code will run on your MySQL Server.
This project runs on Apache Tomcat, so you will need to configure it as well.
Maven is used as the packaging tool, and you need to enable the import all of the dependencies and plugins.
```

File: README.md [hunk 4]
```diff
@@ -43,4 +44,9 @@ login: testAdmin
...
+
+Project was made by Andrii Romash https://github.com/andromash
```
Comment (AnaLanda) [inline README.md]: Edit from older commits:

```
## Idea of the shop
Only authenticated users can use the full functionality of the shop. Authentication is implemented via a filter.
After a user is registered, they are assigned the 'USER' role by default. Authorization is also implemented via a filter.
In the shop prototype presented in this repository, the shop visitors can have only two roles: USER and ADMIN.
##### USER: 
Users can see all of the products that are presented in the shop (except for deleted one - soft deletion is implemented in the project).
Every user has one shopping cart. Users can add products to their cart and then complete order; they can see all of their previous orders and their details.
##### ADMIN:
This role is supposed to be assigned manually by the system administrator. An admin can see all users, delete a user, see all orders, delete an order, add products, and delete products. An admin does not have a cart and cannot place an order.
The project also allows for a combination of the two roles -  if a user has two roles, they can perform all of the actions described for both roles.
If you want to log in as an Admin, open the .../admin/add link in your browser, and a new user with Admin's role will be created with the following login and password:

```

**Review by AnaLanda — COMMENTED** [review]
Looks good overall! Went into a full editing mode to make your text better English-wise) No offense)

**Comment by andromash** [comment]
> Looks good overall! Went into a full editing mode to make your text better English-wise) No offense)

That's awesome! Thank you very much <3

File: README.md [hunk 5]
```diff
@@ -21,26 +20,26 @@ Also, I tried to keep in mind SOLID principles and RBAC
...
+After a user is registered, they are assigned the 'USER' role by default. Authorization is also implemented via a filter.
+In the shop prototype presented in this repository, the shop visitors can have only two roles: USER and ADMIN.
 
-##### USER: 
-can see all of the products,
-that exist in the shop (but not deleted, soft deleting is implemented), every USER has one shopping cart,
- can add product to cart and then complete order, can see all of its orders and details of them.
-##### ADMIN:
+##### USER 
+Users can see all of the products that are presented in the shop (except for deleted one - soft deletion is implemented in the project).
```
Comment (AnaLanda) [inline README.md]:
```suggestion
Users can see all of the products that are presented in the shop (except for deleted ones - soft deletion is implemented in the project).
```

File: README.md [hunk 6]
```diff
@@ -21,26 +20,26 @@ Also, I tried to keep in mind SOLID principles and RBAC
...
+Every user has one shopping cart. Users can add products to their cart and then complete order; they can see all of their previous orders and their details.
```
Comment (AnaLanda) [inline README.md]:
```suggestion
Every user has one shopping cart. Users can add products to their cart and then complete the order; they can see all of their previous orders and their details.
```

File: README.md [hunk 7]
```diff
@@ -21,26 +20,26 @@ Also, I tried to keep in mind SOLID principles and RBAC
...
-
-If you want to enter as Admin, there's address /admin/add  which will add new user with Admin's role
-
-login: testAdmin
+This role is supposed to be assigned manually by the system administrator. An admin can see all users, delete a user, see all orders, delete an order, add products, and delete products. An admin does not have a cart and cannot place an order.
+The project also allows for a combination of the two roles -  if a user has two roles, they can perform all of the actions described for both roles.
+If you want to log in as an Admin, open the .../admin/add link in your browser, and a new user with Admin's role will be created with the following login and password:
 
-password: 1111
+Also, it is predicted that roles can be combined. So if user has two roles - it can do both.
```
Comment (AnaLanda) [inline README.md]:
```suggestion
Also, it is predicted that roles can be combined. So if a user has two roles - they can perform both sets of actions.
```

File: README.md [hunk 8]
```diff
@@ -21,26 +20,26 @@ Also, I tried to keep in mind SOLID principles and RBAC
...
 
 But first of all, there's file init_db.sql which holds sql script for creating appropriate DB.
-So run this code in MySQL on your machine first
+To launch the project, download the project onto your machine and make sure you have MySQL Workbench on it. The file `init_db.sql` holds the SQL script for creating the DB for this project - you need to run the code from this file in MySQL Workbench first. Then you need to change the name and password
+in `ConnectionUtil` so this code will run on your MySQL Server.
+This project runs on Apache Tomcat, so you will need to configure it as well.
+Maven is used as the packaging tool, and you need to enable the import all of the dependencies and plugins.
+
+Project was made by Andrii Romash https://github.com/andromash
```
Comment (AnaLanda) [inline README.md]:
```suggestion
Project created by Andrii Romash https://github.com/andromash
```

**Review by RomanDubovskyi — APPROVED** [review]

**Pull request merged**